tools/wptrunner/wptrunner/executors/executorchrome.py

```diff
@@ -1,9 +1,122 @@
+import os
+import traceback
+
+from six.moves.urllib.parse import urljoin
+
+from .base import WdspecExecutor, get_pages
+from .executorwebdriver import WebDriverProtocol, WebDriverRefTestExecutor, WebDriverRun
+from .protocol import PrintProtocolPart
 from ..webdriver_server import ChromeDriverServer
-from .base import WdspecExecutor, WebDriverProtocol
+
+here = os.path.join(os.path.split(__file__)[0])
+
+
+class ChromeDriverPrintProtocolPart(PrintProtocolPart):
+    def setup(self):
+        self.webdriver = self.parent.webdriver
+        self.runner_handle = None
+
+    def load_runner(self):
+        url = urljoin(self.parent.executor.server_url("http"), "/print_reftest_runner.html")
+        self.logger.debug("Loading %s" % url)
+        try:
+            self.webdriver.url = url
+        except Exception as e:
+            self.logger.critical(
+                "Loading initial page %s failed. Ensure that the "
+                "there are no other programs bound to this port and "
+                "that your firewall rules or network setup does not "
+                r"prevent access.\e%s" % (url, traceback.format_exc(e)))
```

Comment:
I'm not familiar with `\e` here - what is it?

Reply:
I'm thinking it's a typo that got introduced in 2015 and no one noticed until now.